### ISSUE TITLE
Experiment Configuration - Add Annotator Bug

### ISSUE THREAD

**Issue by quhfus** [dice-group/gerbil#128]
Hey Gerbil Team,

There seems to be bug in the current version at the experiment configurations screen. When I add my own annotator (i.e. entering name and correct uri) and click the "Add Annotator Button",  an error message appears quite often that there would be an error with the annotator. 

However, when clicking a second or third time on the "add annotator" button it works fine.

On Gerbil 1.1.4, this bug had never occurred.
Basically, this is only a minor issue since clicking on the button twice "solves" the problem.

Regards

**Comment by TortugaAttack** [comment]
seems to work again. 
Is the problem still occuring? 
If so, can you tell me which version do you use and which browser?

**Comment by chelsea1992** [comment]
Hi,

When I use doser Annotator, it says "too many single errors occur". How can I fix it?

**Comment by TortugaAttack** [comment]
hey,
could you please create a new issue for the problem and i will try to take care of it ;) 
Cheerz

**Comment by chelsea1992** [comment]
Sure

**Comment by RicardoUsbeck** [comment]
Is the original still appearing?

**Comment by TortugaAttack** [comment]
not that i am aware of, it worked for me and i cannot reproduce it.
Thus I will close the issue now.
